.gitignore

```diff
@@ -1,2 +1,2 @@
 node_modules
-.watch
```

Comment:
I would prefer to keep the Makefile and my little watch script in for running my tests automatically.